### PR TITLE
feat(text-field): add dark style mode

### PR DESCRIPTION
**Pull request by ricardogoncalves89**

### CODE REVIEW

File: packages/ui-components/src/components/text-field/text-field.base.scss [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+	// .svg-icon {
+	// 	position: absolute;
+	// 	top: 0;
+	// 	right: $spacing-2x;
+	// 	height: 100%;
+	// 	cursor: pointer;
+	// 	user-select: none;
+	// }
```
Comment (djcouto) [inline packages/ui-components/src/components/text-field/text-field.base.scss:137]: Remove please

**Review by djcouto — APPROVED** [review]
🚀

**Comment by pedrorebelocosta** [comment]
👏  gj

**Review by ricardoarasilva — APPROVED** [review]

File: packages/ui-components/src/globals/globals.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { setMode } from '@stencil/core';
```
Comment (miguelpinto98) [inline packages/ui-components/src/globals/globals.ts:1]: Probably is better to add this option to our readme with an example of application.

File: packages/ui-components/src/globals/globals.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { get } from 'lodash-es';
+import { StyleMode, UIComponentsConfig } from '../types';
+
+let defaultStyleMode: StyleMode = StyleMode.Night;
+
+export const initialize = (userConfig: UIComponentsConfig = {}) => {
+	defaultStyleMode = get(userConfig, 'styleMode', defaultStyleMode);
```
Comment (miguelpinto98) [inline packages/ui-components/src/globals/globals.ts:8]: instead of creating a global variable why we don't put the `StyleMode.Night` at the end?

**Comment by miguelpinto98** [comment]
After all, I liked of this suggestion. Good job @ricardogoncalves89 !

File: packages/react-ui-components/src/stories/Components/TextField.stories.tsx [hunk 4]
```diff
@@ -26,41 +38,67 @@ Default.args = {
...
+	type: EInputFieldType.Text
 };
 
 export const Disabled = TextFieldTemplate.bind({});
 Disabled.args = {
 	...Default.args,
 	label: 'Disabled Text Field',
-	disabled: true
+	disabled: true,
+	state: EValidationState.None,
```
[the remark this answers is not in the document]
Comment (ricardogoncalves89) [reply packages/react-ui-components/src/stories/Components/TextField.stories.tsx:52]: Storybook doesn't read the default value and doesn't select it in the controls

**Review by miguelpinto98 — APPROVED** [review]

**Pull request merged**